src/main/java/mate/academy/bookstore/handler/BookControllerExceptionHandler.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.bookstore.handler;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import mate.academy.bookstore.exception.EntityNotFoundException;
+import org.springframework.dao.DataIntegrityViolationException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.validation.ObjectError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+@ControllerAdvice
+public class BookControllerExceptionHandler {
```

Comment:
sholdn't it be extended from ResponseEntityExceptionHandler?

Reply:
Agree

Reply:
No, it works without it